bigquery/snippets/loadingdata/integration_test.go

```diff
@@ -96,13 +96,13 @@ func TestImportSnippets(t *testing.T) {
 			}
 		})
 		/*
-		t.Run("importJSONWithCMEK", func(t *testing.T) {
-			t.Parallel()
-			tableID := "bigquery_load_table_gcs_json_cmek"
-			if err := importJSONWithCMEK(tc.ProjectID, testDatasetID, tableID); err != nil {
-				t.Errorf("importJSONWithCMEK(%q): %v", testDatasetID, err)
-			}
-		})
+			t.Run("importJSONWithCMEK", func(t *testing.T) {
+				t.Parallel()
+				tableID := "bigquery_load_table_gcs_json_cmek"
+				if err := importJSONWithCMEK(tc.ProjectID, testDatasetID, tableID); err != nil {
+					t.Errorf("importJSONWithCMEK(%q): %v", testDatasetID, err)
+				}
+			})
```

Comment:
Delete?

Reply:
I'd disabled them as the tests were dependent on config not available to me locally due to how CMEK authz works.  Re-enabled all the CMEK related samples in the integration tests.

Reply:
For a separate PR: we should make it so that particular case gets skipped if the config is not available (ideally, all tests pass if you clone the repo and run `go test ./...`.